kopi/assetlist.json

```diff
@@ -0,0 +1,282 @@
+      "name": "kUSD",
+      "display": "kUSD",
+      "symbol": "kUSD",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/kusd.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/kusd.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm",
+        "automations",
+        "arbitrage"
+      ],
+      "socials": {
+        "twitter": "https://x.com/kopi_money",
+        "discord": "https://discord.gg/kopi-money"
+      },
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "A MM deposit receipt of the Stablecoin native to the Kopi blockchain, token is created by the native kopi MM minted by the protocol",
+      "denom_units": [
+        {
+          "denom": "uckusd",
+          "exponent": 0
+        },
+        {
+          "denom": "ckUSD",
+          "exponent": 6
+        }
+      ],
+      "base": "uckusd",
+      "name": "ckUSD",
+      "display": "ckUSD",
+      "symbol": "ckUSD",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/kusd.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/kusd.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm"
+      ],
+      "socials": {
+        "twitter": "https://x.com/kopi_money",
+        "discord": "https://discord.gg/kopi-money"
+      },
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "A MM deposit receipt of the Stablecoin USDC from noble Chain, deposited IBC token on the native kopi MM minted by the protocol",
+      "denom_units": [
+        {
+          "denom": "ucusdc",
+          "exponent": 0
+        },
+        {
+          "denom": "cUSDC",
+          "exponent": 6
+        }
+      ],
+      "base": "ucusdc",
+      "name": "cUSDC",
+      "display": "cUSDC",
+      "symbol": "cUSDC",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/usdc.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/usdc.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm"
+      ],
+      "socials": {
+        "twitter": "https://x.com/kopi_money",
+        "discord": "https://discord.gg/kopi-money"
+      },
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "A MM deposit receipt of the Stablecoin USDT from Injective Chain",
+      "denom_units": [
+        {
+          "denom": "ucusdtinj",
+          "exponent": 0
+        },
+        {
+          "denom": "cUSDT.inj",
+          "exponent": 6
+        }
+      ],
+      "base": "ucusdtinj",
+      "name": "cUSDT.inj",
+      "display": "cUSDT.inj",
+      "symbol": "cUSDT.inj",
```

Comment:
Please define the `traces` (how this derives from USDT.inj (from: ibc/D8A36AE90F20FE4843A8D249B1BCF0CCDDE35C4B605C8DED57BED20C639162D0))